galley/pkg/kube/converter/converter.go

```diff
@@ -84,17 +85,21 @@ func convertJSON(from, to interface{}) error {
 func identity(_ *Config, destination resource.Info, name resource.FullName, _ string, u *unstructured.Unstructured) ([]Entry, error) {
 	var p proto.Message
 	creationTime := time.Time{}
+	var metadata resource.Metadata
 	if u != nil {
 		var err error
 		if p, err = toProto(destination, u.Object["spec"]); err != nil {
 			return nil, err
 		}
 		creationTime = u.GetCreationTimestamp().Time
+		metadata.Labels = u.GetLabels()
```

Comment:
It looks like these will allocate a copy, when labels exists, but it will return nil if there is no entry. Is that the case?